### PR TITLE
many: fix formatting for go1.19

### PR DESCRIPTION
**Pull request by MiguelPires**
Go 1.19 includes some changes to gofmt which intend to make lists and headings clearer when being rendered (https://go.dev/doc/go1.19). These changes are the result of running the new gofmt (version 1.19.4) and manually fixing some of it. Manual fixes were necessary because the new gofmt assumes lines beginning with tabs always begin lists or examples that should have newlines. While this is often true in our codebase, we occasionally also use tabs to indent the lines after a TODO or FIXME prefix or in yaml (e.g., excerpts of a snap.yaml). This meant that a lot of the reformatted comments were broken and had to be fixed manually. In one case, there's yaml in a list item so that's hard to get right.

### CODE REVIEW

File: cmd/snap-seccomp/main_test.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/seccomp/libseccomp-golang"
+	seccomp "github.com/seccomp/libseccomp-golang"
```
Comment (mvo5) [inline cmd/snap-seccomp/main_test.go:33]: Why was that one needed?

**Review by mvo5 — APPROVED** [review]
Thanks and WOAH that looks certainly like a lot of work.

What is nice is that AFAICT go-1.18 and lower will not reformat again (iirc we had that in the past with some go-1.13 fmt changes)

File: cmd/snap-seccomp/main_test.go (hunk 1 above)
Comment (MiguelPires) [reply cmd/snap-seccomp/main_test.go:33]: Ah that was `goimports` ran by a plugin in my editor, not `gofmt`. I thought I had fixed it but must've reverted it. Thanks for spotting it, I'll fix it now

**Comment by MiguelPires** [comment]
> What is nice is that AFAICT go-1.18 and lower will not reformat again (iirc we had that in the past with some go-1.13 fmt changes)

I think so, I tested that and it seems stable. Btw, I changed the doc comment with a yaml snippet in a list (that didn't format the yaml well) to be more readable by removing the `-` list items. In this case it's better to not have the list rather than the yaml formatting IMO

**Pull request merged**